### PR TITLE
Updates keyValueUrl to one that exists

### PR DESCRIPTION
**Pull request by vertein**

### CODE REVIEW

File: angularjs-portal-home/src/main/webapp/js/app-config.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ define(['angular'], function(angular) {
             },
             'notificationsURL' : '/web/samples/notifications.json',
             'groupURL' : '/portal/api/groups',
-            'kvURL' : '/portal/api/keyvalue/'
+            'kvURL' : '/storage/'
```
Comment (timlevett) [inline angularjs-portal-home/src/main/webapp/js/app-config.js]: This was more for mock data than anything else, hmmm, how to handle this... We do overlay it in environments.

Comment (vertein) [inline angularjs-portal-home/src/main/webapp/js/app-config.js]: could leave it null and overlay it?

Comment (timlevett) [inline angularjs-portal-home/src/main/webapp/js/app-config.js]: We could, or just leave it as `/portal/api/keyvalue` for mock dataness

Comment (vertein) [inline angularjs-portal-home/src/main/webapp/js/app-config.js]: Ahhh, I just looked.  I didn't realize there was actual mock data there.  I'll close this and we can overlay.

Comment (timlevett) [inline angularjs-portal-home/src/main/webapp/js/app-config.js]: :+1: for action taken

**Pull request closed without merging**